src/config/chis-tg/config.json

```
@@ -0,0 +1,189 @@
+
+{
+  "domains": [
+    {
+      "friendly": "MoH Togo DEV",
+      "domain": "moh-togo.dev.medicmobile.org"
+    },
+    {
+      "friendly": "MoH Togo UAT",
+      "domain": "moh-togo-uat.dev.medicmobile.org"
+    }
+  ],
+  "contact_types": [
+    {
+      "name": "cg70_supervision",
+      "friendly": "Supervision Area",
+      "contact_type": "cg72_supervision",
+      "user_role": ["supervisor"],
+      "username_from_place": false,
+      "deactivate_users_on_replace": true,
+      "hierarchy": [
+        {
+          "friendly_name": "Sanitary Training",
+          "property_name": "sanitary_training",
+          "contact_type": "cf60_sanitary",
+          "type": "name",
+          "required": true,
+          "level": 1
+        }
+      ],
+      "replacement_property": {
+        "friendly_name": "Affected Supervision Area",
+        "property_name": "replacement",
+        "type": "string",
+        "parameter": ["\\(Supervision Area Name\\)", "\\(SUPERVISION AREA NAME\\)"],
+        "required": true
+      },
+      "place_properties": [
+        {
+          "friendly_name": "Supervision Area Name",
+          "property_name": "name",
+          "type": "generated",
+          "parameter": "{{ contact.last_name }} {{ contact.first_name }} ({{ lineage.sanitary_training }})",
+          "required": false
+        }
+      ],
+      "contact_properties": [
+        {
+          "friendly_name": "Last Name",
+          "property_name": "last_name",
+          "type": "string",
+          "parameter": ["\\(LAST NAME\\)", "\\(LASTNAME\\)", "\\(LAST-NAME\\)"],
+          "required": true
+        },
+        {
+          "friendly_name": "First Name",
+          "property_name": "first_name",
+          "type": "string",
+          "parameter": ["\\(Chw First Name\\)", "\\(Chw Firstname\\)"],
```

```suggestion
          "parameter": ["\\(*First*\\)"],
```

Remove anything in brackets including `first`?